packages/gatsby-theme-api-docs/src/hooks/use-type-locations.js

```diff
@@ -9,6 +9,8 @@ const buildPageSlug = (page) => {
 };
 
 const convertComponentInMdxToTypeLocations = (data) =>
+  // TODO refactor this inefficient access pattern into a plain loop
```

Comment:
@timonrey do you think it makes sense to do this separately or here?  I vote for separately but soon, you have routine now in this refactor.

Reply:
separate issue exists, I removed this line